crates/api_models/src/payments.rs

```diff
@@ -1563,6 +1592,7 @@ pub struct Metadata {
     pub order_category: Option<String>,
 
     /// Redirection response coming in request as metadata field only for redirection scenarios
+    #[schema(value_type = Option<RedirectResponse>)]
```

Comment:
You should be able to remove the `value_type` specification.

Reply:
Okay